pkg/jx/cmd/preview.go

```diff
@@ -432,18 +432,15 @@ func (o *PreviewOptions) Run() error {
 	}
 
 	values := config.PreviewValuesConfig{
-		ExposeController: &config.ExposeController{
-			Config: config.ExposeControllerConfig{
-				Domain: domain,
-			},
-		},
+		ExposeController: o.HelmValuesConfig.ExposeController,
```

Comment:
`ExposeController` can be `nil` in the `HelmValueConfig`. Please could you add a check for this? It would be great if you could add a small test for this scenario.